client/index.js

```diff
@@ -1,15 +1,22 @@
 import { Provider } from 'react-redux';
 import ReactDOM from 'react-dom';
 import React from 'react';
+import { 
+	BrowserRouter as Router, 
+	Redirect 
+} from 'react-router-dom';
 
 import App from 'containers/App';
 import store from 'store';
 
-import 'index.html';
-
 ReactDOM.render(
   <Provider store={store}>
-    <App />
+    <Router>
```

Comment:
we definitely want just one component nested under Router. 
We should be able to define IndexRoute in Router config. 
Not exactly sure about React Router v4, but I hope they haven't changed that